src/components/CompareResults/beta/RevisionHeader.tsx

```diff
@@ -38,10 +39,37 @@ const styles = {
   }),
 };
 
-function createTitle(header: RevisionsHeader) {
-  return header.test === '' || header.suite === header.test
-    ? header.suite
-    : `${header.suite} ${header.test}`;
+function createTitle(
+  header: RevisionsHeader,
+  docsURL: string,
+  isLinkSupported: boolean,
+) {
+  if (isLinkSupported) {
+    return header.test === '' || header.suite === header.test ? (
```

Comment:
Could you move `header.test === '' || header.suite === header.test` into a constant?

Reply:
I was wondering what to name it, maybe `isTestNamePresent` ?